src/components/cylc/Tree.vue

```diff
@@ -106,6 +113,11 @@ export default {
           this.activeCache.add(treeItem)
         }
       }
+    },
+    clearCaches () {
+      this.treeItemCache.clear()
+      this.activeCache.clear()
+      this.expandedCache.clear()
```

Comment:
These are cache instances that were not being cleared before. So they were increasing a lot over time, and leaving retained objects in the heap, preventing GC to collect them.